src/main/java/uk/gov/hmcts/ccd/RestTemplateConfiguration.java

```diff
@@ -95,7 +95,11 @@ private HttpClient getHttpClient(final int timeout) {
         cm.setMaxTotal(maxTotalHttpClient);
         cm.closeIdleConnections(maxSecondsIdleConnection, TimeUnit.SECONDS);
         cm.setDefaultMaxPerRoute(maxClientPerRoute);
-        cm.setValidateAfterInactivity(validateAfterInactivity);
+
+        // Temporally 30 value until we can tune the real value then It will be set in the ENV variables.
+        cm.setValidateAfterInactivity(10);
```

Comment:
This value is in ms. 10 seems too short

Reply:
The default is 15 . I think that we can try 2

Reply:
Default is 2000ms from what I can see. Yes, it seems reasonable